ssc/cmod_battery.cpp

```diff
@@ -1303,360 +1267,299 @@ battstor::battstor(const battstor& orig){
+        { SSC_INPUT,        SSC_NUMBER,      "system_use_lifetime_output",                 "Lifetime simulation",                                     "0/1",        "0=SingleYearRepeated,1=RunEveryYear",   "Lifetime",        "?=0",                   "BOOLEAN",                              "" },
+        { SSC_INPUT,        SSC_NUMBER,      "analysis_period",                            "Lifetime analysis period",                                "years",      "The number of years in the simulation", "Lifetime",        "system_use_lifetime_output=1","",                               "" },
+        { SSC_INPUT,        SSC_NUMBER,      "en_batt",                                    "Enable battery storage model",                            "0/1",        "",                     "BatterySystem",                      "?=0",                    "",                               "" },
+        { SSC_INOUT,        SSC_ARRAY,       "gen",										  "System power generated",                                  "kW",         "",                     "System Output",                             "",                       "",                               "" },
+        { SSC_INPUT,		SSC_ARRAY,	     "load",			                              "Electricity load (year 1)",                               "kW",	        "",				        "Load",                             "",	                      "",	                            "" },
+        { SSC_INPUT,		SSC_ARRAY,	     "crit_load",			                      "Critical electricity load (year 1)",                      "kW",	        "",				        "Load",                             "",	                      "",	                            "" },
+        { SSC_INOUT,        SSC_NUMBER,      "capacity_factor",                            "Capacity factor",                                         "%",          "",                     "System Output",                             "?=0",                    "",                               "" },
+        { SSC_INOUT,        SSC_NUMBER,      "annual_energy",                              "Annual Energy",                                           "kWh",        "",                     "System Output",                      "?=0",                    "",                               "" },
+
+        // other variables come from battstor common table
+        var_info_invalid };
 
 extern var_info vtab_fuelcell_output[];
 
 void process_messages(shared_ptr<battstor> batt, compute_module* cm)
 {
     if (!batt)
         return;
-    message dispatch_messages = batt->dispatch_model->get_messages();
-    message thermal_messages = batt->thermal_model->get_messages();
+//    message dispatch_messages = batt->dispatch_model->get_messages();
```

Comment:
Can these be removed or do the messages need to be restored?

Reply:
I removed the `message` class since I wanted to consolidate those into the logger. So will remove this.